basic/src/tags/components/TagLabel.tsx

```diff
@@ -0,0 +1,46 @@
+import { ClassNameValue, twJoin } from "tailwind-merge";
+import { Tag } from "wasp/entities";
+
+type TagLabelSize = "md" | "sm" | "tiny";
+
+interface TagLabelProps {
+  tag: Pick<Tag, "id" | "color" | "name">;
+  isActive: boolean;
+  size?: TagLabelSize;
+}
+
+export function TagLabel({ tag, isActive, size = "md" }: TagLabelProps) {
+  return (
+    <span
+      className={twJoin(
+        "inline-block rounded-full border font-mono font-semibold text-black",
+        sizeStyles[size],
+      )}
+      style={
+        isActive
+          ? {
+              backgroundColor: tag.color,
+              borderColor: "black",
+            }
+          : {
+              background: `repeating-linear-gradient(
+              45deg,
+              hsl(from ${tag.color} h s l / 0.50),
+              hsl(from ${tag.color} h s l / 0.50) 4px,
+              hsl(0, 0%, 0%, 0.30) 4px,
+              hsl(0, 0%, 0%, 0.30) 6px
+              )`,
+              borderColor: "black",
+            }
+      }
+    >
+      {tag.name}
+    </span>
+  );
+}
```

Comment:
This will be a wild change, I played around with the design:
```suggestion
export function TagLabel({ tag, isActive, size = "md" }: TagLabelProps) {
  return (
    <span
      className={twJoin(
        "inline-flex items-center gap-2 rounded-full border-2 font-mono font-semibold text-black",
        sizeStyles[size],
      )}
      style={{
        backgroundColor: isActive ? tag.color : "transparent",
      }}
    >
      {tag.name}
      {size === "md" && (
        <span
          className="inline-block h-3 w-3 rounded-full"
          style={{
            backgroundColor: tag.color,
            border: isActive ? "2px solid black" : "1px solid transparent",
          }}
        ></span>
      )}
    </span>
  );
}
```


https://github.com/user-attachments/assets/652da02b-c454-42e0-8e37-1b8add2f446b

The idea was making it less colorful so it's less distracting from the main content. I went with the colored circle - maybe you'll like it.

Reply:
Damn I actually really like the little colored circle. 
But not the black outline circle when it's active is a bit weird to me. But removing it makes the content change size.

How about this? Reminds me of tag icon:
<img width="363" alt="image" src="https://github.com/user-attachments/assets/8f1d4932-e75d-40e5-af66-db712b2fd3e9" />
<img width="86" alt="image" src="https://github.com/user-attachments/assets/d9be3b5f-ed60-48bd-a5bb-568e46b43aa2" />